model-setup-examples/basic-elevon.md

```diff
@@ -0,0 +1,22 @@
+## Basic Elevon
+
+Elevons are control surfaces that combine the functions of the elevator and the aileron.
```

Comment:
@schwabe, so the suggestion here is to go more into depth on the theory of control surfaces, etc?
@RCdiy , you are right, this isn't ready to be merged.  I wanted to put this out so people would have a chance to put their two cents in before I spent a ton of time writing what only I thought was needed.  I do appreciate your sentiments, however.